test/functional/plugin/lsp_spec.lua

```diff
@@ -5022,6 +5022,35 @@ describe('LSP', function()
       eq('initialize', result.method)
     end)
 
+    it('can connect to lsp server via rpc.connect using hostname', function()
```

Comment:
almost identical to the test above, would you mind factoring out the code into a function?
```suggestion
    it('connects to lsp server via rpc.connect(hostname)', function()
```